cmd/scaffold.go

```diff
@@ -672,6 +680,11 @@ func (fnb *FlowNodeBuilder) Component(name string, f func(*FlowNodeBuilder) (mod
 	return fnb
 }
 
+func (fnb *FlowNodeBuilder) PreInit(f func(node *FlowNodeBuilder)) *FlowNodeBuilder {
```

Comment:
These functions help customize the initialization. In the case of the unstaked access node, it lets setting the `local` for the node so that it is attempted to be initialized later from the protocol state.

Reply:
We may need a code comment that clarifies the two code paths here:
- one where the initialization runs from the protocol state, i.e. the normal path
- one where that info is not in the protocol state accessible to the node, and therefore needs to be set through other means (the unstaked node path)